demo/SFSW2_project_descriptions.R

```diff
@@ -206,10 +206,19 @@ opt_input <- list(
       #   - Harmonized World Soil Database: 1-km re-gridded; data expected
       #     at project_paths[["dir_ex_soil"]], "CONUSSoil", "output", "albers")
       "ExtractSoilDataFromCONUSSOILFromSTATSGO_USA", 0,
+      # Soil texture
+      #   - Soil data from penn state for the United States; data expected
+      #     at: project_paths[["dir_ex_soil"]], "Isric", GriddedGlobalV5
+      "ExtractSoilDataFromIsricSoilGrid_Global_250m", 0,
```

Comment:
ISRIC is all capitals (https://www.isric.online/about), see use in, e.g., line 2016 "ExtractSoilDataFromISRICWISEv12_Global"